LondonBritishLibrary/orient/BLorient708.xml

```diff
@@ -37,26 +37,26 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <msContents>
                         <summary/>
                         <msItem xml:id="ms_i1">
-                            <locus target="#5r"/>
-                            <title type="complete" ref="LIT1893Martyr">The Acts of Fasiladas</title>
+                            <locus from="5r" to="91r"/>
+                            <title type="complete" ref="LIT1893Martyr">The Acts of Fāsiladas</title>
                             <textLang mainLang="gez"/>
                             
                             
                         </msItem>
                         
                         <msItem xml:id="ms_i1.2">
-                            <locus target="#91r"/>
-                            <title type="complete" ref="LIT1893Martyr#Taamer">The Miracle of Fasiladas</title>
+                            <locus from="91r" to="95v"/>
+                            <title type="complete" ref="LIT1893Martyr#Taamer">The Miracle of Fāsiladas</title>
                             <textLang mainLang="gez">ኃይለ፡ ተአምር፡ ዘገብረ፡ ቅዱስ፡ ፋሲለደስ፡ በደሴት፡ እንተ፡ ትሰመይ፡ ምጽሌ።</textLang>
                             
                             
                         </msItem>
                         <msItem xml:id="ms_i1.3">
-                            <locus target="#91r"/>
-                            <title type="complete" ref="LIT1893Martyr#hymn">The Miracle of Fasiladas</title>
+                            <locus from="95v" to="101v"/><!-- stanzas 1 to 40 (l. 1) -->
```

Comment:
I would maybe include the commented out information on the stanzas explicitely in a `<note>`, so that app users can also have a chance of learning this (except if you are planing further work or a transcription of this manuscript...).

Reply:
Wasn't sure how much leeway there was with `<note>`s, but it is "noted" 😁.

Reply:
Yes, it's best used sparingly...but I feel that this is an appropriate case, instead of hiding relevant information in a comment (which I used to do very often)